new/src/sections/events/events.scss

```diff
@@ -0,0 +1,122 @@
+@import "./mainEventItem.scss";
+@import "./eventItem.scss";
+
+.events {
+  width: 100%;
+  height: 100%;
+  padding: 3rem 0;
+}
+
+.container,
+.container-fluid,
+.container-xxl,
+.container-xl,
+.container-lg,
+.container-md,
+.container-sm {
+  width: 100%;
+  padding-right: var(--bs-gutter-x, 0.75rem);
+  padding-left: var(--bs-gutter-x, 0.75rem);
+  margin-right: auto;
+  margin-left: auto;
+  height: fixed;
+  display: grid;
+}
```

Comment:
Please start your class names with `events`. Once imported in `App.scss`, these class names may be confused with in other components.